### DISCUSSION TITLE
What about Ombi when a user reseted his Jellyfin's password

### DISCUSSION THREAD

**Discussion by Tonomis** [hrfee/jfa-go#92] [Q&A]
How do you manage Ombi accounts when users have changed/reseted their password ?

**Comment by hrfee** [comment]
I'll work this into the link password reset method ("Use reset link instead of PIN" in Settings), I don't think there's a way to do it otherwise.

**Reply by Tonomis** [reply]
So nice !
And when they just changed their password into Jellyfin ?

**Reply by hrfee** [reply]
There isn't a way to get the password from Jellyfin so the user would have to change it themselves. This will only set the password for Jellyfin & Ombi to the PIN number.

**Reply by hrfee** [reply]
Just pushed in af61549bf155cf0a3b156426f1df3b8decfbd1d6, it took a while because I thought i'd found a bug in Ombi that broke this and another feature, but by the time i'd written a bug report the issue stopped happening lol.
